apps/webapp/README.md

```diff
@@ -0,0 +1,18 @@
+# Webapp
+
+To start your Phoenix app:
+
+  * Install dependencies with `mix deps.get`
+  * Start Phoenix endpoint with `mix phoenix.server`
+
+Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
+
+Ready to run in production? Please [check our deployment guides](http://www.phoenixframework.org/docs/deployment).
```

Comment:
I believe we shouldn't add this here because the project is not *production ready yet*